aws/resource_aws_s3_bucket_test.go

```diff
@@ -663,6 +663,49 @@ func TestAccAWSS3Bucket_UpdateAcl(t *testing.T) {
 	})
 }
 
+func TestAccAWSS3Bucket_UpdateGrant(t *testing.T) {
```

Comment:
Please add acceptance tests for converting from `acl` to `grant` and for switching from `grant` to `acl`.

Reply:
done